src/components/time-selector/TimeSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { checkboxArray } from './Checkboxes'
+import { ICheckbox } from './TimeSelectorInterfaces'
+import { TimeSelectorCard } from '../card/TimeSelectorCard'
+
+export const TimeSelector = () => {
+
+    const [cards, setCards] = useState<string[]>([])
+    const [checkboxes, setCheckboxes] = useState<ICheckbox[]>(checkboxArray)
+
+    const getCards = () => {
+        return ( 
+            cards?.map(name => {
```

Comment:
`name ` value actually contains a single card value. I think we should use `card` instead of `name` here.